injector/src/test/java/edu/ucr/cs/riple/injector/OnFieldInjectionTest.java

```diff
@@ -62,6 +62,35 @@ public void fieldNullableSimple() {
         .start();
   }
 
+  @Test
+  public void fieldNullableArray() {
```

Comment:
This is not a good name for this test. Unit test names should demonstrate what feature we are testing exactly.

Reply:
Removed this test case